trulens_eval/examples/snowflake_demo/common_ui.py

```diff
@@ -251,6 +254,17 @@ def configure_model(
                 if model_config.use_rag != st.session_state[USE_RAG_KEY]:
                     st.session_state[USE_RAG_KEY] = model_config.use_rag
 
+                if model_config.retrieval_filter != st.session_state[RETRIEVAL_FILTER_KEY]:
```

Comment:
Shouldn't we set the state after `model_config.retrieval_filter` has been assigned by the slider?

Reply:
Fixed!